apis/ec2/v1alpha2/register.go

```diff
@@ -15,9 +15,9 @@ limitations under the License.
 */
 // +kubebuilder:object:generate=true
 // +groupName=ec2.aws.crossplane.io
-// +versionName=v1alpha1
+// +versionName=v1alpha2
```

Comment:
Then you would keep the version inside of `manualv1alpha1` as `v1alpha1`. CRDs should not change for existing `VPCCIDRBlock`, you will just need to change imports in the controller / client / etc.